msupply-foundation/notify
# 255 parameters in events table

Closes #255

# 👩🏻‍💻 What does this PR do?
 

Adds the parameters and query results as JSON to the bottom of the notification event log. Also adds the recipient name to the `To` field. This should help to improve troubleshooting.

Persists the tera context with the notification event during the render and enqueue step of the notification event.

![Screenshot 2023-11-21 at 11 20 44 AM](https://github.com/msupply-foundation/notify/assets/55115239/43cf2b34-1ddd-4d2c-834c-5935e9d7d970)


# 🧪 How has/should this change been tested?


- Trigger a new notification event - set a scheduled event for a minute from now, and wait for it to complete.
- View the event details for the new notification event
- See any parameters or query results in the JSON field at the bottom of the event details
- Recipient name is part of the `To` field

## 💌 Any notes for the reviewer?


## 📃 Documentation

_No user facing changes_

or
